mypy_django_plugin/lib/fullnames.py

```diff
@@ -30,7 +31,14 @@
     BASE_MANAGER_CLASS_FULLNAME,
 }
 
-RELATED_FIELDS_CLASSES = {FOREIGN_KEY_FULLNAME, ONETOONE_FIELD_FULLNAME, MANYTOMANY_FIELD_FULLNAME}
+RELATED_FIELDS_CLASSES = frozenset(
+    (
+        FOREIGN_OBJECT_FULLNAME,
+        FOREIGN_KEY_FULLNAME,
+        ONETOONE_FIELD_FULLNAME,
```

Comment:
Both `ForiegnKey` and `OneToOneField` are subclasses of `ForeignObject`, so these could be removed now.

`ManyToManyField` has a separate inheritance hierarchy though.

Reply:
Good catch!